GPU/Common/FragmentShaderGenerator.cpp

```diff
@@ -462,6 +462,12 @@ bool GenerateFragmentShader(const FShaderID &id, char *buffer, const ShaderLangu
 		WRITE(p, "}\n");
 	}
 
+	if (compat.bitwiseOps && enableColorTest) {
+		p.C("ivec3 unpackIVec3(highp uint x) {\n");
+		p.C("  return ivec3(x & 0xFF, (x >> 8) & 0xFF, (x >> 16) & 0xFF);\n");
```

Comment:
Should this be `uvec3` instead?  We only use it as uvec3.  Not sure if some GLES driver might not like the type conversion (paranoia.)

-[Unknown]

Reply:
yeah, it probably should. I'll change it.